doc/whats_new/v0.20.rst

```diff
@@ -116,6 +116,10 @@ Classifiers and regressors
   :class:`linear_model.BayesianRidge` for weighted linear regression.
   :issue:`10111` by :user:`Peter St. John <pstjohn>`.
 
+- :class:'dummy.DummyClassifier' and :class:'dummy.DummyRegresssor' now
+  only requires X to be an object with finite length or shape.
+  :issue:'9832' by :user:'Vrishank Bhardwaj <vrishank97>'.
```

Comment:
You should use ` instead of ', please follow what others' are doing and check the result from Circle CI.

Reply:
Oops.